content/en/docs/collector/custom-collector.md

```diff
@@ -0,0 +1,190 @@
+component code.
+
+The OpenTelemetry Community developed a tool called [OpenTelemetry Collector
+builder][ocb] (or `ocb` for short) to assist people in assembling their own
+distribution, making it easy to build a distribution that includes their custom
+components along with components that are publicly available.
+
+As part of the process the `builder` will generate the Collector's source code,
+which you can use to help build and debug your own custom components, so let's
+get started.
+
+## Step 1 - Install the builder
+
+The `ocb` binary is available as a downloadable asset within the
+[OpenTelemetry Collector Releases Page](https://github.com/open-telemetry/opentelemetry-collector/releases)
+within the Collector's GitHub project. You will find the list of assets at the
+bottom of the page.
+
+v0.53.0 is the latest and the assets are named based on OS and chipset, so
+download the one that fits your configuration.
+
+The binary has a pretty long name, so you can simply rename it to `ocb`; and if
+you are running Linux or macOS, you will also need to provide execution
+permissions for the binary.
+
+Open your terminal and type the following commands to accomplish both
+operations:
+
+```cmd
+mv ocb_0.53.0_darwin_amd64 ocb
+chmod 777 ocb
+```
+
+To make sure the `ocb` is ready to be used, go to your terminal and type
+`./ocb help`, and once you hit enter you should have the following output:
+
+```
+OpenTelemetry Collector distribution builder (dev)
+
+Usage:
+  builder [flags]
+  builder [command]
+
+Available Commands:
+  completion  Generate the autocompletion script for the specified shell
+  help        Help about any command
+  version     Version of opentelemetry-collector-builder
+
+Flags:
+      --config string            config file (default is $HOME/.otelcol-builder.yaml)
+      --description string       A descriptive name for the OpenTelemetry Collector distribution (default "Custom OpenTelemetry Collector distribution")
+      --go string                The Go binary to use during the compilation phase. Default: go from the PATH
+  -h, --help                     help for builder
+      --module string            The Go module for the new distribution (default "go.opentelemetry.io/collector/cmd/builder")
+      --name string              The executable name for the OpenTelemetry Collector distribution (default "otelcol-custom")
+      --otelcol-version string   Which version of OpenTelemetry Collector to use as base (default "0.42.0")
+      --output-path string       Where to write the resulting files (default "/var/folders/86/s7l1czb16g124tng0d7wyrtw0000gn/T/otelcol-distribution3618633831")
+      --skip-compilation         Whether builder should only generate go code with no compile of the collector (default false)
+      --version string           The version for the OpenTelemetry Collector distribution (default "1.0.0")
+
+Use "builder [command] --help" for more information about a command.
+```
```

Comment:
Can we drop the code block showing the help output? Folks should know if the command is running even without this. I'm worried that this output will become stale too quickly. (But more importantly, I don't think it supports the intent of this page.)

Reply:
agreed, I  removed it and adjusted the content to reflect it.